gsdk/src/signer/utils.rs

```diff
@@ -105,18 +103,18 @@ impl Inner {
             }
 
             match status {
-                Future | Ready | Broadcast(_) | Retracted(_) => (),
-                InBlock(b) => {
+                Validated | Broadcasted { .. } | NoLongerInBestBlock => (),
```

Comment:
it simply means the current best block doesn't include this transaction, we do not need to handle it here since we already have logs for it:

https://github.com/paritytech/subxt/blob/d44941fb1bdb1b46ef8537dd7511654facc3f763/subxt/src/backend/unstable/mod.rs#L513

and 

https://github.com/paritytech/subxt/blob/d44941fb1bdb1b46ef8537dd7511654facc3f763/subxt/src/backend/unstable/mod.rs#L527